src/search-field/SearchField.ts

```diff
@@ -85,6 +87,12 @@ export class SearchField extends OmniFormElement {
         this.value = input?.value;
     }
 
+    _blurOnEnter(e: KeyboardEvent) {
```

Comment:
Is this used?

Reply:
Added event listener to call this function accordingly and tested locally.